src/main/java/seedu/address/commons/core/AliasMapping.java

```diff
@@ -62,6 +62,24 @@ private AliasMapping(AliasMapping aliasMapping) {
         this.mapping = new HashMap<>(aliasMapping.mapping);
     }
 
+    /**
+     * Sets the mapping to the mapping of the specified AliasMapping object.
+     *
+     * @param aliasMapping The specified AliasMapping object.
+     */
+    public void setAliasMapping(AliasMapping aliasMapping) {
+        this.mapping = aliasMapping.mapping;
```

Comment:
Hmm, could this instead make a copy of alias mappings? Otherwise undo may be more difficult to support.

Reply:
Done!